internal/commands/user/disable_test.go

```diff
@@ -0,0 +1,324 @@
+			ID:         "user-3",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeUserPassword}},
+			Type:       "type-1",
+			Data:       map[string]interface{}{"email": "user-3@test.com"},
+		},
+		{
+			ID:         "user-4",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeAPIKey}},
+			Type:       "type-1",
+			Data:       map[string]interface{}{"name": "name-4"},
+		},
+		{
+			ID:         "user-5",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeCustomToken}},
+			Type:       "type-3",
+		},
+	}
+	for _, tc := range []struct {
+		description    string
+		outputs        []userOutput
+		expectedOutput string
+	}{
+		{
+			description:    "should show indicate no users to disable",
+			outputs:        []userOutput{},
+			expectedOutput: "01:23:45 UTC INFO  No users to disable\n",
+		},
+		{
+			description: "should show 1 failed user, command: user disable",
+			outputs: []userOutput{
+				{user: testUsers[0], err: errors.New("client error")},
+			},
+			expectedOutput: strings.Join(
+				[]string{
+					"01:23:45 UTC INFO  Provider type: User/Password",
+					"  Email            ID      Type    Enabled  Details     ",
+					"  ---------------  ------  ------  -------  ------------",
+					"  user-1@test.com  user-1  type-1  true     client error",
+					"",
+				},
+				"\n",
+			),
+		},
+		{
+			description: "should show 2 failed users",
+			outputs: []userOutput{
+				{user: testUsers[0], err: nil},
+				{user: testUsers[1], err: errors.New("client error")},
+				{user: testUsers[2], err: nil},
+				{user: testUsers[3], err: errors.New("client error")},
+				{user: testUsers[4], err: nil},
+			},
+			expectedOutput: strings.Join(
+				[]string{
+					"01:23:45 UTC INFO  Provider type: User/Password",
+					"  Email            ID      Type    Enabled  Details     ",
+					"  ---------------  ------  ------  -------  ------------",
+					"  user-2@test.com  user-2  type-2  true     client error",
+					"  user-1@test.com  user-1  type-1  false                ",
+					"  user-3@test.com  user-3  type-1  false                ",
+					"01:23:45 UTC INFO  Provider type: ApiKey",
+					"  Name    ID      Type    Enabled  Details     ",
+					"  ------  ------  ------  -------  ------------",
+					"  name-4  user-4  type-1  true     client error",
+					"01:23:45 UTC INFO  Provider type: Custom JWT",
+					"  ID      Type    Enabled  Details",
+					"  ------  ------  -------  -------",
+					"  user-5  type-3  false           ",
+					"",
+				},
+				"\n",
+			),
+		},
+	} {
+		t.Run(tc.description, func(t *testing.T) {
+			out, ui := mock.NewUI()
+			cmd := &CommandDisable{outputs: tc.outputs}
+
+			assert.Nil(t, cmd.Feedback(nil, ui))
+			assert.Equal(t, tc.expectedOutput, out.String())
+		})
+	}
+}
+
+func TestUserDisableRow(t *testing.T) {
+	for _, tc := range []struct {
+		description      string
+		authProviderType realm.AuthProviderType
+		output           userOutput
```

Comment:
something's off with this test case struct here, I think ultimately you want to remove `output` field here?

currently, it doesn't look like `tc.authProviderType`, but if you keep that and add `data map[string]interface{}` and `outputErr error` fields, then the `userOutput` struct can be built inside of the test function (rather than here in the `tc` initialization)